src/server/controllers/signInControllers.ts

```diff
@@ -417,6 +415,8 @@ export const oktaIdxApiSignInController = async ({
 			}
 		}
 
+		// Otherwise use password sign in
```

Comment:
if the usePasswordSignIn flag is not set but there's no passcode in the request body then it would attempt password sign in ? is this the desired behaviour ?

Reply:
That's right. It only attempts to use passcodes if it sees the `passcode` in the request params, which is just a hidden flag saying to use the passcode flow rather than the password flow